index.js

```diff
@@ -67,6 +81,31 @@ module.exports = function (src, dest, opts) {
 				resolve(true);
 			});
 
+			var progressEnabled = progress.emitter.listeners('progress').length > 0;
+			if (progressEnabled) {
```

Comment:
If you still think it is worth, I've just found [`emitter.listenerCount(eventName)`](https://nodejs.org/api/events.html#events_emitter_listenercount_eventname).